share/eds/byzantine/byzantine.go

```diff
@@ -41,8 +42,12 @@ func NewErrByzantine(
 		dah.ColumnRoots,
 		dah.RowRoots,
 	}[errByz.Axis]
+	axisType := rsmt2d.Row
+	if errByz.Axis == rsmt2d.Row {
+		axisType = rsmt2d.Col
+	}
```

Comment:
This seems wrong. Why? To make test work after breaking Validate error?

Reply:
There is a comment out of the scope of Github (2 lines above) specifying that we need to collect proofs against ortogonal axis. Anyway, refactored it a bit more to get rid of [][][]byte. Looks cleaner now